examples/reshare/src/orchestrator/actor.rs

```diff
@@ -220,9 +214,6 @@ where
                 // If we're not in the committee of the latest epoch we know about and we observe another
                 // participant that is ahead of us, send a message on the orchestrator channel to prompt
                 // them to send us the finalization of the epoch boundary block for our latest known epoch.
-                if rate_limiter.check_key(&from).is_err() {
```

Comment:
I suppose we would technically be doing more work here now by checking if we need the finalization if the peer is hammering us with messages...

Reply:
Actually, changed my mind. I think we should offer a `pre_send` that checks_key/returns something we can call send on (which bypasses RL).

Reply:
Added an issue for consideration / discussion, since this'd be a fairly invasive change: #2468